docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,233 @@ Example output:
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
+
+    ```bash
+    sudo apt-get install tpm2-tools
+    ```
+
+    [Building from source](https://tpm2-tools.readthedocs.io/en/latest/INSTALL/)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Execute command in the terminal:
+
+    ```bash
+    sudo tpm2_nvdefine 0x1500016
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output should look similar as below:
+
+    ```bash
+    nv-index: 0x1500016
+    ```
+
+## TPM011.001 NVUNDEFINE Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that NVUNDEFINE function works properly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
+
+    ```bash
+    sudo apt-get install tpm2-tools
+    ```
+
+    [Building from source](https://tpm2-tools.readthedocs.io/en/latest/INSTALL/)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Execute command in the terminal:
+
+    ```bash
+    sudo tpm2_nvundefine 0x1500016
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output should be empty. If region was undefined or error occured output
+    should look like this:
+
+    ```bash
+    WARNING:esys:src/tss2-esys/api/Esys_NV_ReadPublic.c:309:Esys_NV_ReadPublic_Finish() Received TPM Error 
+    ERROR:esys:src/tss2-esys/esys_tr.c:209:Esys_TR_FromTPMPublic_Finish() Error NV_ReadPublic ErrorCode (0x0000018b) 
+    ERROR:esys:src/tss2-esys/esys_tr.c:320:Esys_TR_FromTPMPublic() Error TR FromTPMPublic ErrorCode (0x0000018b) 
+    ERROR: Esys_TR_FromTPMPublic(0x18B) - tpm:handle(1):the handle is not correct for the use
+    ERROR: Failed to read the public part of NV index 0x1500016
+    ERROR: Unable to run tpm2_nvundefine
+    ```
+
+## TPM012.001 CREATE Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that create functionw works as expected.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
+
+    ```bash
+    sudo apt-get install tpm2-tools
+    ```
+
+    [Building from source](https://tpm2-tools.readthedocs.io/en/latest/INSTALL/)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Execute command in the terminal:
+
+    ```bash
+    sudo tpm2_create -C primary.ctx -u obj.pub -r obj.priv
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output should look similar as below:
```

Comment:
And what does this output contains? How is anyone supposed to determine whether a feature is working or not?

Reply:
Updated